src/main/java/io/appium/java_client/touch/offset/PointOption.java

```diff
@@ -0,0 +1,56 @@
+package io.appium.java_client.touch.offset;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.String.format;
+import static java.util.Optional.ofNullable;
+
+import org.openqa.selenium.Point;
+
+import java.util.Map;
+
+public class PointOption extends AbstractPositionOption<PointOption> {
+
+    private static final String ERROR_MESSAGE_TEMPLATE = "%s coordinate value should be equal or greater than zero";
+
+    /**
+     * It creates a built instance of {@link PointOption} which takes x and y coordinates.
+     * This is offset from the upper left corner of the screen.
+     *
+     * @param xOffset is x value.
+     * @param yOffset is y value.
+     * @return a built option
+     */
+    public static PointOption coordinates(int xOffset, int yOffset) {
+        return new PointOption().position(xOffset, yOffset);
+    }
+
+    /**
+     * It defines x and y coordinates.
+     * This is offset from the upper left corner of the screen.
+     *
+     * @param xOffset is x value.
+     * @param yOffset is y value.
+     * @return self-reference
+     */
+    @Override
+    public PointOption position(int xOffset, int yOffset) {
```

Comment:
why not AbsoluteOffsetOption ?

Reply:
I think it is not completely correct. When we say `absolute offset` it means `coordinates` or 'point' on the screen.